docs/source/configuration.rst

```diff
@@ -46,11 +46,11 @@ Consider:
 
 .. code-block:: shell
 
-   tartufo --config myconfig.toml scan-local-repo .
+   tartufo --config myconfig.toml scan-local-repo <target_directory>
```

Comment:
I believe if we leave it as it is(with the <target_directory> instead of .) we can be generic with a single example in both the scenarios where users perform Tartufo scan from a different location or be in the repositories location to start a scan.

I am open to suggestions if you still think otherwise.